doc/quickstart.md

```diff
@@ -40,262 +83,206 @@ Let's unpack things here:
 - We give a name our entity, by default it will be assumed that the
   SQL table it resides in is eponymous, when it is not the case, a
   tuple of `[entity-name table-name]` can be provided
-- *Ident* fields are unique in the database and can be used to
-  retrieve a single record.
+- We declare a list of fields known to exist in that table.
 
 With this, simple queries can be performed:
 
 ```clojure
-(query env :account [:account/name :account/state])
+(query env ::account/account [::account/name ::account/state])
+;; or to fetch all default fields:
+(query env ::account/account)
 
 ;; =>
 
-[#:account{:name "a0" :state "active"}
- #:account{:name "a1" :state "active"}
- #:account{:name "a2" :state "suspended"}]
+[#::account{:name "a0" :state :active}
+ #::account{:name "a1" :state :active}
+ #::account{:name "a2" :state :suspended}]
 ```
 
 Idents can also be looked up:
 
 ```clojure
-(query env [:account/id 0] [:account/name :account/state])
+(query env [::account/id 0] [::account/name ::account/state])
 
 ;; =>
 
-#:account{:name "a0" :state "active"}
+#::account{:name "a0" :state :active}
 ```
 
 Notice how the last query yielded a single value instead of a collection.
 It is expected that idents will yield at most a single value (as a corollary,
 idents should only be used for database fields which enforce this guarantee).
 
-While this works, our schema can be improved in two ways:
+Also notice how there was no prior mention of `::account/id`
+
+### Infering schemas from specs
+
+A first concrete improvement we can bring to the schema build step when
+an `s/keys` spec is available for our entity is to infer most of the schema
+from it:
+
+``` clojure
+(make-schema
+  (from-spec ::account/account))
```

Comment:
`entity` was a primary construct of the previous version of SEQL.
I expect it to be no longer used at all in this next iteration, being replaced by `from-spec`.
However, most of the documentation mentions `entity`.

SEQL supposes a `one-to-one` mapping between an entity and a table and the concept of `entity` by not being present on the schema makes that fact unclear in my opinion.

I would prefer having: `entity-from-spec` instead of only `from-spec`.